table.csv

```diff
@@ -549,7 +549,7 @@ iscc,                           softhash,       0xcc01,         draft,      ISCC
 zeroxcert-imprint-256,          zeroxcert,      0xce11,         draft,      0xcert Asset Imprint (root hash)
 nonstandard-sig,                varsig,         0xd000,         deprecated, Namespace for all not yet standard signature algorithms
 bcrypt-pbkdf,                   multihash,      0xd00d,         draft,      Bcrypt-PBKDF key derivation function
-es256k,                         varsig,         0xd0e7,         draft,      ES256K Siganture Algorithm (secp256k1)
+es256k,                         varsig,         0xd0e7,         draft,      ES256K Signature Algorithm (secp256k1)
```

Comment:
This feels like a definite typo, and as such, i'm hoping all of the fields are free form as opposed to tied to something...